core/templates/pages/exploration-player-page/services/player-position.service.ts

```diff
@@ -40,6 +40,10 @@ export class PlayerPositionService {
     this.onChangeCallback = callback;
   }
 
+  /**
+   * This function is used to find the name of the current state.
+   * @return {string} a string that shows the name of the current state.
```

Comment:
Maybe just 'The name of the current state'?

Reply:
Fixed.